src/utils/utils.hpp

```diff
@@ -90,7 +90,7 @@ const std::string default_command_map_name = "command_map";
  *
  * @note This will have to be manually changed after the release
  */
-const std::string current_host_app_version = "2.1.0";
+const std::string current_host_app_version = "3.0.0";
```

Comment:
XMOS projects have a habit of duplicating the version number in multiple locations.  I hope host_xvf_control has largely avoided this tendency.  However, it might be worth searching for `2.1.0` before merging just to be sure.

Reply:
That's how I found this one! That's luckily the only place. I'll suggest adding Brennan's version checker into the Jenkinsfile in a new VFH issue.